app/logic/searchUsers.py

```diff
@@ -15,6 +15,8 @@ def searchUsers(query):
         for participant in results:
             if participant not in resultsDict:
                 resultsDict[f"{participant.firstName} {participant.lastName} ({participant.username})"] = f"{participant.firstName} {participant.lastName} ({participant.username})"
+                if object=="true":
+                    resultsDict[f"{participant.username} phoneNumber"] = participant.phoneNumber
```

Comment:
This isn't very intuitive to me. If object is true, we add a key with a phone number in it? It probably makes sense for how you are using it, but searchUsers is a very generic function. The use cases should be clear in both comments and code.